weave/trace/trace_sentry.py

```diff
@@ -58,7 +67,9 @@ def __init__(self) -> None:
 
         self.dsn = SENTRY_DEFAULT_DSN
 
-        self.hub: sentry_sdk.hub.Hub | None = None
+        self.hub: sentry_sdk.Hub | None = None
```

Comment:
this is really weird to me that we need to manage both - can you explain the approach and reasoning behind this PR in the description?